app.js

```diff
@@ -4,15 +4,35 @@ const app = express()
 const port = process.env.PORT || 3000
 
 const mysql = require('mysql2')
-const connection = mysql.createConnection({
-  host : process.env.DATABASE_HOST,
-  database: process.env.DATABASE_NAME,
-  user : process.env.DATABASE_USERNAME,
-  password : process.env.DATABASE_PASSWORD,
-  ssl : {
-      rejectUnauthorized: true
-  }
-});
+let connection = {};
+
+/*
+  ---- Note: You only need one connection object in the if/else statement ----
+
+  OPTION 1:
+    Use if you're using connection strings in your app (see docs)
+    https://docs.planetscale.com/tutorials/connect-nodejs-app#using-a-connection-string
+
+  OPTION 2:
+    Use if you're connecting with client certificates via CLI (see docs)
+    https://docs.planetscale.com/tutorials/connect-nodejs-app#using-client-certificates-with-the-cli
+    
+  Feel free to remove the if/else statement once you decide which you'll use
+*/
+
+if (process.env.DATABASE_HOST) {
+  connection = mysql.createConnection({
+    host : process.env.DATABASE_HOST,
+    database: process.env.DATABASE_NAME,
+    user : process.env.DATABASE_USERNAME,
+    password : process.env.DATABASE_PASSWORD,
+    ssl : {
+        rejectUnauthorized: true
+    }
+  });
+} else {
+  connection = mysql.createConnection(process.env.DATABASE_URL)
```

Comment:
I'm not sure this will work. If we are using client certificates, how do we know it's in the exact format that we need for this package. Specifically, what used to be `ssl=true` at the end of the string now has to be `ssl={}` with the mysql2 package. Has this been tested with client certificates?